plugins/blur-up/ls.blur-up.js

```diff
@@ -18,6 +18,28 @@
 	window : 0, function(window, document, lazySizes) {
 	'use strict';
 
+	var lazySizesCfg;
+
+	(function(){
+		var prop;
+
+		var blurUpDefaults = {
+			blurUpClass: 'ls-blur-up-img',
+			blurUpLoadingClass: 'ls-blur-up-is-loading',
+			blurUpInviewClass: 'ls-inview',
+			blurUpLoadedClass: 'ls-blur-up-loaded',
+			blurUpLoadedOriginalClass: 'ls-original-loaded'
+		};
+
+		lazySizesCfg = window.lazySizesConfig || window.lazysizesConfig || {};
```

Comment:
We do not rely on the global `lazySizesConfig`. You can simply copy it from `lazySizes.cfg` + copy it at the start where you declare the variable -> `var lazySizesCfg  = lazySizes.cfg;`

Reply:
@aFarkas done